Coding standards/PHP coding standards.md

```diff
@@ -24,8 +24,26 @@ namespace Inf_Theme;
 
 Every folder which holds classes constitutes a subnamespace.
 
+Be aware that WordPress 'lives' in a global namespace. So, when using functions and classes from the WordPress core, it's necessary to put a slash in front of class or function name, so that it's called from the global namespace. For example, calling `WP_Query` inside your class should be done like sources
+
+```php
+$query = new \WP_Query( $arguments );
```

Comment:
Could we maybe add a phpcs rule that throws a warning / error when not doing this?